app/connectors/ClaimToAdjustPoaConnector.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import config.MicroserviceAppConfig
+import connectors.httpParsers.ClaimToAdjustPoaHttpParser._
+import models.claimToAdjustPoa.ClaimToAdjustPoaRequest
+import models.claimToAdjustPoa.ClaimToAdjustPoaResponse.{ClaimToAdjustPoaResponse, ErrorResponse}
+import play.api.Logger
+import play.api.http.Status.INTERNAL_SERVER_ERROR
+import play.api.libs.json.Json
+import uk.gov.hmrc.http.client.HttpClientV2
+import uk.gov.hmrc.http.{HeaderCarrier, StringContextOps}
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+
+class ClaimToAdjustPoaConnector @Inject() ( val appConfig: MicroserviceAppConfig,
+                                            val http: HttpClientV2)
```

Comment:
:question: - just wondering if we have to start migration work around rest of the httpClients we have so far in our BE  to use *V2?

Reply:
Tech debt story created => https://jira.tools.tax.service.gov.uk/browse/MISUV-7715